### PR TITLE
regenerate DEBio tutorial II

### PR DESCRIPTION
**Pull request by isaacsas**
Also fixes an error when Weaving the footer in html pages when `pkg.old` is `nothing`.

### CODE REVIEW

File: tutorials/models/04-diffeqbio_II_networkproperties.jmd [hunk 1]
```diff
@@ -411,11 +411,11 @@ oprob = ODEProblem(rn, u₀, tspan, p)
 
 We are now ready to solve the problem and plot the solution. Since we have
 essentially generated a method of lines discretization of the diffusion equation
-with a discontinuous initial condition, we'll use an A-L stable implicit ODE
-solver, `KenCarp4`, and plot the solution at a few times:
+with a discontinuous initial condition, we'll use an A stable implicit ODE
```
Comment (ChrisRackauckas) [inline tutorials/models/04-diffeqbio_II_networkproperties.jmd]:
```suggestion
with a discontinuous initial condition, we'll use an A-L stable implicit ODE
```

Comment (ChrisRackauckas) [reply tutorials/models/04-diffeqbio_II_networkproperties.jmd]: we should probably still mention the L-stability, which is the most important property there.

Comment (isaacsas) [reply tutorials/models/04-diffeqbio_II_networkproperties.jmd]: `Rodas5` is L-stable too? (I just ask as the docs, http://docs.juliadiffeq.org/latest/solvers/ode_solve.html, only say that it is A-stable.)

Comment (ChrisRackauckas) [reply tutorials/models/04-diffeqbio_II_networkproperties.jmd]: yeah it's L-stable, and IIRC it has good B-stability properties as well. We should probably update the docs on that.

Comment (isaacsas) [reply tutorials/models/04-diffeqbio_II_networkproperties.jmd]: That's good to know! (Haven't read much about those methods before...) I'll put the L-stability back in and regenerate everything tomorrow when I have a chance.

**Pull request merged**